lib/ruby_lsp/requests/formatting.rb

```diff
@@ -71,6 +85,7 @@ def run
       sig { returns(T.nilable(String)) }
       def formatted_file
         case @formatter
+        when "none"
```

Comment:
I don't think we need to add a branch for `none` if it just returns `nil`.

Reply:
Removed, it was a remnant from a previous approach.